tests/api_test/api_test.cpp

```diff
@@ -597,15 +597,15 @@ bind_tailcall_test(_In_ struct bpf_object* object)
 
 #define SOCKET_TEST_PORT 0x3bbf
 
-void
-send_traffic(IPPROTO protocol, bool is_ipv6)
+void send_traffic(IPPROTO protocol, bool is_ipv6)
```

Comment:
Is there any change in this file actually relevant to this PR?
All I spotted was unrelated whitespace changes.

Reply:
another \format-code.ps1 side effect.

Reply:
format-code.ps shouldn't be doing this.  Please file an issue to track.